etc/sudoers.d/spinnaker

```diff
@@ -0,0 +1,2 @@
+# User rules for spinnaker
+spinnaker ALL=(ALL) NOPASSWD:ALL
```

Comment:
If we change this to the following, it will only allow `sudo` for the `packer` command:
`spinnaker ALL=(ALL) NOPASSWD: /usr/bin/packer`